test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs

```diff
@@ -23,11 +23,12 @@ public class GivenThatIWantANewAppWithSpecifiedType : TestBase
         [InlineData("C#", "web", true)]
         [InlineData("C#", "mvc", true)]
         [InlineData("C#", "webapi", true)]
-        [InlineData("F#", "console", false)]
-        [InlineData("F#", "classlib", false)]
-        [InlineData("F#", "mstest", false)]
-        [InlineData("F#", "xunit", false)]
-        [InlineData("F#", "mvc", true)]
+        // Uncomment the test below once https://github.com/dotnet/netcorecli-fsc/issues/92 is fixed.
+        //[InlineData("F#", "console", false)]
```

Comment:
We shouldn't completely turn these tests off.

Instead, we should just disable them on the RIDs where it isn't supported.

See [https://github.com/dotnet/cli/blob/rel/1.0.1/test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs#L33-L38](https://github.com/dotnet/cli/blob/rel/1.0.1/test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs#L33-L38)